app/controllers/api/v1/brands_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::BrandsController < ApplicationController
+  def index
+    # @pizza_brands = Pizza.brandOnly 
+    render json: Brand.all
+  end
+
+  def show
+    render json: Brand.find(params[:id]).pizzas
+  end
+end
```

Comment:
make sure to add controller tests for each of these endpoints 

while not required, start to consider adding a serializer